backend/profiles/migrations/0001_initial.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.1.7 on 2019-03-23 16:31
+
+from django.db import migrations
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+def create_beatles(apps, schema_editor):
```

Comment:
A few things:

1. Why `create_beatles`?
2. Why are we doing this through a migration? This will only run once. Let's sit down for a little bit tonight and we can talk about adding an execution step to the container run command, rather than doing this via migration.
3. What happened to the load_data command?

Reply:
1.  `create_beatles` should be changed to `create_profiles`.
2.  I'm unclear why we would want to add the users more than once, though maybe I just don't understand what we're trying to do.  Can you please explain?
3.  You can add users with loaddata, but their passwords won't be hashed in the db, which prevents you from actually being able to log in.